lib/iris/analysis/__init__.py

```diff
@@ -1,4 +1,4 @@
-# (C) British Crown Copyright 2010 - 2016, Met Office
+# (C) British Crown Copyright 2010 - 2017, Met Office
```

Comment:
Now I'm confused the travis build from #2290 includes `lib/iris/analysis/__init__.py` as one of it's failures
```
======================================================================

ERROR: test_license_headers (iris.tests.test_coding_standards.TestLicenseHeaders)

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/home/travis/build/SciTools/iris/lib/iris/tests/test_coding_standards.py", line 345, in test_license_headers

    raise ValueError('There were license header failures. See stdout.')

ValueError: There were license header failures. See stdout.

-------------------- >> begin captured stdout << ---------------------

The file header at lib/iris/analysis/__init__.py is out of date. The last commit was in 2017, but the copyright states it was 2016.

The file header at lib/iris/analysis/_interpolate_backdoor.py is out of date. The last commit was in 2017, but the copyright states it was 2016.

The file header at lib/iris/analysis/interpolate.py is out of date. The last commit was in 2017, but the copyright states it was 2016.

--------------------- >> end captured stdout << ----------------------
```
but that file hasn't been touched since #2264 which was merged before the new year
https://github.com/SciTools/iris/commits/master/lib/iris/analysis/__init__.py